src/SWP/Bundle/ContentListBundle/Doctrine/ORM/ContentListRepository.php

```diff
@@ -33,4 +33,12 @@ public function findByType(string $type): array
             ->getResult()
         ;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function flush()
```

Comment:
repository should not be able to update, it should do only remove and add when it comes to these operations. This is object manager responsibility. We should also remove persist from EntityRepository in StorageBundle because it just adds object to the manager and it doesn't affect the repository changes at that time.